src/main/java/hudson/plugins/ansicolor/action/ColorizedAction.java

```diff
@@ -29,14 +29,17 @@
 import hudson.model.Run;
 import hudson.plugins.ansicolor.AnsiColorMap;
 
+import java.util.Optional;
 import java.util.UUID;
+import java.util.logging.Logger;
 
 import static hudson.plugins.ansicolor.action.ActionNote.TAG_ACTION_BEGIN;
 
 /**
  * Action for issuing commands to ColorConsoleAnnotator
  */
 public class ColorizedAction extends InvisibleAction {
+    private static final Logger LOGGER = Logger.getLogger(ColorizedAction.class.getName());
```

Comment:
Looks like code uses `LOG` and `LOGGER` in other places, for the future maybe make them consistent. 

I prefer `LOGGER`.

Reply:
> I prefer `LOGGER`.

Agreed